Rakefile

```diff
@@ -5,5 +5,6 @@ require "bundler/gem_tasks"
 task default: [:test]
 
 task :test do
-  exec "cd javascript && yarn run test"
+  system "bin/ruby_test"
+  system "cd javascript && yarn run test"
```

Comment:
What does system return? Will it exit with a non zero code if the command fails?

Reply:
@hopsoft 
```
➜ rake test
Run options: --seed 23909

# Running:

F

Failure:
StimulusReflexGeneratorTest#test_creates_named_controller_and_reflex_files [/Users/andrew.mason/work/OSS/stimulus_reflex/test/generators/stimulus_reflex_generator_test.rb:14]:
Expected file "app/reflexes/test_reflex.rb" to not exist, but does


bin/test Users/andrew.mason/work/OSS/stimulus_reflex/test/generators/stimulus_reflex_generator_test.rb:8

No value provided for required arguments 'name'
..No value provided for required arguments 'name'
..No value provided for required arguments 'name'
.

Finished in 0.014597s, 411.0434 runs/s, 1233.1301 assertions/s.
6 runs, 18 assertions, 1 failures, 0 errors, 0 skips
yarn run v1.19.0
$ yarn run mocha --require @babel/register
$ /Users/andrew.mason/work/OSS/stimulus_reflex/javascript/node_modules/.bin/mocha --require @babel/register


  attributeValue
    ✓ returns falsey for no arguments
    ✓ returns falsey for empty array
    ✓ returns expected attribute value for array with single value
    ✓ returns expected attribute value for array with multiple values
    ✓ returns expected attribute value for array with multiple values that include whitespace
    ✓ returns expected attribute value for array with mixed values

  attributeValues
    ✓ returns empty array for no arguments
    ✓ returns expected attribute values for single value
    ✓ returns expected attribute values for multiple values
    ✓ returns expected attribute values for multiple values with whitespace padding

  extractElementAttributes
    ✓ returns expected attributes for empty anchor
    ✓ returns expected attributes for anchor
    ✓ returns expected attributes for textarea
    ✓ returns expected attributes for textbox
    ✓ returns expected attributes for unchecked checkbox
    ✓ returns expected attributes for checked checkbox

  matchingControllerName
    ✓ returns expected controller name for reflex
    ✓ returns expected controller name abbreviated for event + reflex


  18 passing (40ms)

✨  Done in 0.91s.```